tests/functional/z_cluster/cluster_expansion/test_add_capacity_lso_multiple_device_classes.py

```
@@ -0,0 +1,205 @@
+
+        Args:
+            pvc_factory: Fixture for creating PVCs.
+            pod_factory: Fixture for creating Pods.
+            bucket_factory: Fixture for creating object-store buckets.
+            rgw_bucket_factory: Fixture for creating RGW buckets.
+
+        """
+        device_sets = get_all_device_sets()
+        if len(device_sets) < 2:
+            logger.warning(
+                "Skipping: requires multiple device classes " "(found %d device set)",
```

Ruff (0.15.14)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)